frontend/src/utils/admin-utils.ts

```diff
@@ -50,7 +51,10 @@ export const getChartAdminBoundaryParams = (
 ): AdminBoundaryParams => {
   const { serverLayerName, chartData } = layer;
 
-  const { levels, url, fields: datasetFields } = chartData!;
+  const { levels, url: chartUrl, fields: datasetFields } = chartData!;
+
+  // Take in chartUrl if provided, otherwise use default CHART_API_URL
+  const url = chartUrl || CHART_API_URL;
```

Comment:
cc @wadhwamatic this is where we select the URL from layers.json, or default to CHART_API_URL otherwise